# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openlibraries/compile-fix.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur openlibraries-0.4.0-orig/src/openmedialib/plugins/avformat/avformat_plugin.cpp openlibraries-0.4.0/src/openmedialib/plugins/avformat/avformat_plugin.cpp
--- openlibraries-0.4.0-orig/src/openmedialib/plugins/avformat/avformat_plugin.cpp	2008-02-26 18:31:24.000000000 +0100
+++ openlibraries-0.4.0/src/openmedialib/plugins/avformat/avformat_plugin.cpp	2008-02-26 18:38:39.000000000 +0100
@@ -10,6 +10,10 @@
 #include <openpluginlib/pl/pcos/isubject.hpp>
 #include <openpluginlib/pl/pcos/observer.hpp>
 
+#ifndef INT64_C
+#define INT64_C(x)	x ## LL
+#endif // INT64_C
+
 #ifdef WIN32
 #include <windows.h>
 #endif // WIN32
@@ -1917,7 +1921,7 @@
 {
 	public:
 		// filter_type overloads
-		explicit avformat_resampler_filter::avformat_resampler_filter(const opl::wstring &)
+		explicit avformat_resampler_filter(const opl::wstring &)
 			: filter_type()
 			, prop_output_channels_(pcos::key::from_string("channels"))
 			, prop_output_sample_freq_(pcos::key::from_string("frequency"))
@@ -2012,7 +2016,7 @@
 			}
 		}
 
-		frame_type_ptr avformat_resampler_filter::fetch()
+		frame_type_ptr fetch()
 		{
 			// Get the filter input
 			input_type_ptr input = fetch_slot( );
